src/ansys/fluent/core/services/api_upgrade.py

```diff
@@ -44,7 +44,4 @@ def __exit__(self, exc_type, exc_value, exc_tb) -> None:
                 print(
                     "The following solver settings object method could also be used to execute the above command:"
                 )
-                if int(self._version) >= 251:
-                    print(f"<solver_session>.settings.{journal_str[len('solver.'):]}")
-                else:
-                    print(f"<solver_session>.{journal_str[len('solver.'):]}")
+                print(f"<solver_session>.{journal_str[len('solver.'):]}")
```

Comment:
Are we adding a test?

Reply:
This is a display message and has no tests. I'll try to add something.

Reply:
Done, Thank you.